src/CargoOrder.java

```diff
@@ -1,4 +1,8 @@
 package traingame;
 
 public record CargoOrder(Product product, City destination, int payment) {
+	@Override
+	public String toString() {
+		return String.format("%-15s %-20s %-5s%n", product.name().toString(), destination.name().toString(), payment);
```

Comment:
product.name() and destination.name() are already strings

Reply:
OK, I"ll drop the extra toString()s then.